src/slash_commands/chombo.rs

```diff
@@ -107,29 +108,8 @@ impl ChomboCommand {
         subcommand: &ApplicationCommandInteractionDataOption,
         chombot: &Chombot,
     ) -> SlashCommandResult {
-        let user_option = subcommand
-            .options
-            .iter()
-            .find(|option| option.name == CHOMBO_ADD_SUBCOMMAND_USER_OPTION)
-            .unwrap()
-            .resolved
-            .as_ref()
-            .unwrap();
-        let user = match user_option {
-            ApplicationCommandInteractionDataOptionValue::User(user, _) => user,
-            _ => panic!("Invalid option value"),
-        };
-
-        let description = subcommand
-            .options
-            .iter()
-            .find(|option| option.name == CHOMBO_ADD_SUBCOMMAND_DESCRIPTION_OPTION)
-            .unwrap()
-            .value
-            .as_ref()
-            .unwrap()
-            .as_str()
-            .unwrap();
+        let (user, _) = get_user_option(&subcommand.options, CHOMBO_ADD_SUBCOMMAND_USER_OPTION).ok_or("Missing user")?;
```

Comment:
I was thinking if we could benefit from non-`&str` error structures, but I guess because this is strictly user-facing error message, it's okay to keep it that way.